lib/v2/ecust/maintainer.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
     '/jxjy/news': ['jialinghui'],
     '/yjs': ['shengmaosu'],
+    '/jwc/:category?': ['lxl66566'],
```

Comment:
Insert new entries in alphabetical order.
```suggestion
    '/jwc/:category?': ['lxl66566'],
    '/yjs': ['shengmaosu'],
```